it/mail/chrome/messenger/accountCreationModel.properties

```diff
@@ -3,11 +3,11 @@
 
 
 # readFromXML.js
-no_emailProvider.error=Il file XML non contiene la configurazione di un account di posta.
+no_emailProvider.error=Il file XML di configurazione non contiene la configurazione dell'account e-mail.
 outgoing_not_smtp.error=Il server in uscita deve essere di tipo SMTP
 
 # verifyConfig.js
-cannot_login.error=Impossibile autenticarsi sul server. La configurazione, il nome utente o la password potrebbero essere errati.
+cannot_login.error=Impossibile autenticarsi sul server. La configurazione, il nome utente o la password probabilmente sono errati.
```

Comment:
CORRECTIONS NEEDED. I prefer the original version as it sounds more natural. Please revert to the following sentence:  

-cannot_login.error=Impossibile autenticarsi sul server. La configurazione, il nome utente o la password potrebbero essere errati.